src/emissions_harmonization_historical/constants_5000.py

```diff
@@ -225,10 +229,16 @@
 
 # Commit from https://github.com/IAMconsortium/common-definitions
 # to use
+
 # previous: COMMON_DEFINITIONS_COMMIT = "95b5f2c9fb62e32a4d08fe2ffc5b4a6ff246ad2d"
-COMMON_DEFINITIONS_COMMIT = "f2536b68fd52a81fda792c7d3547b9a60c868041"
+# previous: COMMON_DEFINITIONS_COMMIT = "f2536b68fd52a81fda792c7d3547b9a60c868041"
+# previous: COMMON_DEFINITIONS_COMMIT = "f2536b68fd52a81fda792c7d3547b9a60c868041"
+# updated to always grab most recent commit hash from repo
+COMMON_DEFINITIONS_COMMIT = get_latest_commit_hash("IAMconsortium", "common-definitions")
```

Comment:
```suggestion
COMMON_DEFINITIONS_COMMIT = get_latest_commit_hash("IAMconsortium", "common-definitions", fallback_commit="cc69ed0a415a63c7ce7372d5a36c088d9cbee055")
```

Let's make it explicit what the fallback is (just so it's easier to figure out what happens e.g. if one of us runs this without an internet connection)

Reply:
it currently wouldn't work without an internet connection. 
this here was a fallback in case something doesn't work with retrieving the most recent commit hash, so i selected a hardcoded commit. but it might be better to fall back to using whatever local version of the repo is available?